### DISCUSSION TITLE
Is it possible to access the SFP Module via Mikrotik?

### DISCUSSION THREAD

**Discussion by kingsvell** [Anime4000/RTL960x#257] [Q&A]
I bouht recently a Mikrotik RB760iGS and i used a hp switch for configure the module. On the MikroTik i didnt find the answer to access the module. Is it possible?

**Comment by Anime4000** [comment]
for this, you need use ROS 7.15beta4
![image](https://github.com/Anime4000/RTL960x/assets/1908715/2d1d1627-94eb-4850-ba05-1dc6d2d2986d)

- [x] **Ignore RX LOS** (screenshot taken from another RU group)
![image](https://github.com/Anime4000/RTL960x/assets/1908715/58fc2093-d956-4868-a0fc-5d731b791404)

this way, you can config even no fiber attached

**Comment by longthanhtran** [comment]
Assuming your stick use IP address 192.168.1.1, you will need to assign an IP address on your mikrotik sfp interface with the same subnet, e.g 192.168.1.22/24. Then make sure the interface is assigned to WAN group so default firewall rule with masquerade rule will allow you to access from your LAN to the stick.
